packaging/amazon-linux-ami/ecs-init.spec

```diff
@@ -149,6 +149,9 @@ if [ -e %{running_semaphore} ]; then
 fi
 
 %changelog
+* Fri May 26 2017 Adnan Khan <adnkha@amazon.com> - 1.14.2-1
+- Cache Agent version 1.14.2
+- Add functionality for region based agent updates
```

Comment:
There should be an entry here about running agent with userns=host when Docker is running with userns-remap enabled.